templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -734,7 +698,7 @@ Resources:
         - StackId: !Select [2, !Split [/, !Ref AWS::StackId]]
           Prefix: !FindInMap [SampleAppProperties, OotbRepoPrefix, Name]
           Suffix: !Sub
-            - '${SampleAppName}-${Namespace}'
+            - 'tanzu-java-web-app-workload-${Namespace}'
```

Comment:
*nit*: One thing to consider, for things we want to reference more than once in a template, is to introduce a mapping.

An example for such a mapping can be [seen here](https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/main/templates/aws-tap-entrypoint-existing-vpc.template.yaml#L534-L546), and how to use it can be [seen there](https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/main/templates/aws-tap-entrypoint-existing-vpc.template.yaml#L2241-L2247)

We could even rename `Versions` to `Globals` and do something like:

```
Mappings:
  Globals:   #! previously 'Versions'
    current:
      TAPVersion: 1.4.1
      EKSVersion: '1.24'
      ...
      sampleAppName: tanzu-java-web-app-workload
      ...
```

Or maybe @satya-dillikar knows of a better approach on how to have something like "template-global variables" without using parameters, which would expose those to users -- which we don't want.

But yeah, this is just spit-balling, not sure if that's worth it, and def. not blocking.

Reply:
okay that made sense and i added it into the sample app conffig mapping. feel free to take a look if it all flows right

Reply:
All other custom variables methods are tedious. What you have here with **Mappings** is the simplest approach.  Let us use this only.